src/Html/HtmlParser.fs

```diff
@@ -275,13 +273,15 @@ module internal HtmlParser =
         { Attributes : (CharList * CharList) list ref
           CurrentTag : CharList ref
           Content : CharList ref
+          HasFormattedParent: bool ref
```

Comment:
An alternative is that this can be a `mutable HasFormattedParent` to avoid the `ref`, but I don't think that's terribly important.